hs_collection_resource/templates/pages/collectable_resources_modal.html

```diff
@@ -66,6 +68,7 @@ <h4 class="modal-title">Collectable Resources</h4>
                                     </td>
                                 {% endwith %}
                                 <td>
+                                    <!-- TODO:DRC #5030 user_permission tag creates between 1-3 queries per resource -->
```

Comment:
user_permission tag generates up to 3 queries per resource
https://github.com/hydroshare/hydroshare/blob/5030-collection-performance/hs_core/templatetags/hydroshare_tags.py#L26
but I don't see a better way...

Maybe we can just remove this entirely from the UI...

Reply:
@devincowan Yeah that's an expensive attribute to display. I agree that we should remove it from UI. If we had to display it then I may be able to get it computed more efficiently.